homeworks/modosandras/homework-02/fictive_character.py

```diff
@@ -0,0 +1,11 @@
+name = input("Név: ")
```

Comment:
nem hiba, de a .strip().capitalize()-t akár itt is meghívhatod, és akkor nem kell ezért egy külön sor kód (5.sor)